jest.config.js

```diff
@@ -0,0 +1,204 @@
+/*
+ * For a detailed explanation regarding each configuration property and type check, visit:
+ * https://jestjs.io/docs/configuration
+ */
+
+module.exports = {
+  // All imported modules in your tests should be mocked automatically
+  // automock: false,
+
+  // Stop running tests after `n` failures
+  // bail: 0,
+
+  // The directory where Jest should store its cached dependency information
+  // cacheDirectory: "/private/var/folders/fk/c3y07g0576j8_2s9m01pk4qw0000gn/T/jest_dx",
+
+  // Automatically clear mock calls, instances and results before every test.
+  // This does not remove any mock implementation that may have been provided,
+  // so we disable it.
+  // clearMocks: true,
+
+  // Indicates whether the coverage information should be collected while executing the test
+  collectCoverage: true,
+
+  // An array of glob patterns indicating a set of files for which coverage information should be collected
+  collectCoverageFrom: ['./src/**/*.ts'],
```

Comment:
Should this be `app/**/*.js`?

Reply:
Indeed!

Reply:
Ended up restricting it to `lib.js` - i'm not sure if there are ambitions to consider covering the web ui parts for now.